pages/watches-support.json

```diff
@@ -78,14 +81,21 @@
       "features": [
         { "name":"Display", "status":"good" },
         { "name":"Touch", "status":"good" },
-        { "name":"Microphone", "status":"bad" },
         { "name":"Bluetooth", "status":"good" },
         { "name":"Haptics", "status":"good" },
+        { "name":"Tilt-to-Wake", "status":"bad" },
+        { "name":"Ambient Light", "status":"good" },
+        { "name":"Always-on-Display", "status":"good" },
+        { "name":"Dual-Display", "status":"partial" },
+        { "name":"Microphone", "status":"bad" },
+        { "name":"Speaker", "status":"unknown" },
```

Comment:
The speaker doesn't work yet on `catfish`. That's future talk: https://github.com/AsteroidOS/meta-smartwatch/pull/109